nemo/collections/tts/data/data_utils.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
Copyright year shouldn't change. Year corresponds to when file is created not recent edit year

Reply:
is there any guidance on this? I heard differently though.

Reply:
discussed offline. reverted because "You only need to replace the year when you create a new file, while keeping the year as-is even you modify the file."